### ISSUE TITLE
poorly made vm image

### ISSUE THREAD

**Issue by frankli0324** [mrideout/GRFICSv3#1]
the provided ova was not so poorly made that it frustrates me to the hell when taking a course which instructed us to experiment on them.
why rc.local when you have systemd?? why run services in **scripts**????? WHY???
Is this ***Industrial Grade Code***??

**Comment by frankli0324** [comment]
Also, I implore you to try docker

**Comment by Scarecrow928** [comment]
Indeed, it only works on Windows and you have to install virtualbox. make a [docker version of this repo](https://github.com/Scarecrow928/docker-GRFICSv3), maybe eazier to setup.
